components/Map/Map.tsx

```diff
@@ -134,8 +168,8 @@ function MobileTooltip({ tInfo }: { tInfo: MunicipalityTapInfo }) {
           color: colorTheme.newColors.white, height: 14, width: 14, marginRight: 4,
         }}
       />
-      <span style={{ textDecoration: 'underline' }}>{`${tInfo.mData.name}`}</span>
-      {`: ${tInfo.mData.formattedDataPoint}`}
+      <span style={{ textDecoration: 'underline' }}>{`${tInfo.mData.name}`}:</span>
+      <span style={{ color: dataPointColor, fontWeight: 'bold' }}>{`${tInfo.mData.formattedDataPoint}%`}</span>
```

Comment:
Please run "npm run lint" to get a list of small nitpicks that will need to be addressed.